smartapps/smartthings/hue-connect.src/hue-connect.groovy

```diff
@@ -661,10 +665,63 @@ def locationHandler(evt) {
 
 def doDeviceSync(){
 	log.trace "Doing Hue Device Sync!"
+
+	// Check if state.updating failed to clear
+	if (state.lastUpdateStarted < (now() - 20 * 1000) && state.updating) {
+		state.updating = false
+		log.warn "state.updating failed to clear"
+	}
+
 	convertBulbListToMap()
 	poll()
```

Comment:
We do because even if hub polls, it only sends us an answer back when data changes unless we explicitly do a poll()